.github/workflows/beta-release.yml

```
@@ -0,0 +1,87 @@
+          # However, the user asked for the version name to end in -beta.
+          # Gradle handles the app version name.
+          # For the git tag, if we want it to match, we should append -beta here since pubspec doesn't have it.
+          git tag -a v${{ steps.get_version.outputs.version }}-beta -m "Beta Release v${{ steps.get_version.outputs.version }}-beta"
+          git push origin v${{ steps.get_version.outputs.version }}-beta
+
+      - name: Rename APK
+        run: |
+          mv build/app/outputs/flutter-apk/app-beta-release.apk build/app/outputs/flutter-apk/mybudget-beta-v${{ steps.get_version.outputs.version }}-beta+${{ steps.get_version.outputs.build }}.apk
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.8)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)